### PR TITLE
Delete product model 2.0.9 patch

### PR DESCRIPTION
**Pull request by Delphineray**

### CODE REVIEW

File: content/md/products-super-power/enrich-products-variants.md [hunk 1]
```diff
@@ -163,3 +163,34 @@ The list of versions are displayed in the `History` tab.
...
+It's not possible to mass delete product models from the product grid, because the delete of a product model could delete in cascade lots of children product models and variant products.
+:::
+
+## From the product form
+1.  Open the product form for a product model
+1.  Click on the `...` in the top right corner of the screen
+1.  Select `Delete` in the dropdown
+1.  A confirmation pop-in opens
+  ![Add new variant](../img/Products_ProductModel_Delete.png)
+1.  Then `Delete` to confirm you want to delete the product model and its children
```
Comment (NolwennP) [inline content/md/products-super-power/enrich-products-variants.md]: Click on `Delete`

File: content/md/products-super-power/enrich-products-variants.md [hunk 2]
```diff
@@ -163,3 +163,34 @@ The list of versions are displayed in the `History` tab.
...
+# Delete a product model
+
+You can delete a product model from the product grid or the product form.
+
+## From the product grid
+1.  Hover your mouse over the product model line in the grid, then the delete button (a trash icon) will be displayed
+1.  Click on the `Delete` button
+1.  A confirmation pop-in opens
+  ![Add new variant](../img/Products_ProductModel_Delete.png)
+1.  Then `Delete` to confirm you want to delete the product model and its children
```
Comment (NolwennP) [inline content/md/products-super-power/enrich-products-variants.md]: Click on `Delete`

File: content/md/products-super-power/enrich-products-variants.md (hunk 1 above)
Comment (Delphineray) [reply content/md/products-super-power/enrich-products-variants.md]: ok

File: content/md/products-super-power/enrich-products-variants.md (hunk 2 above)
Comment (Delphineray) [reply content/md/products-super-power/enrich-products-variants.md]: ok

**Pull request merged**